src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -95,7 +95,6 @@ public void toModelType_invalidAddress_throwsIllegalValueException() {
         assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
     }
 
-    @Disabled
     @Test
     public void toModelType_nullAddress_throwsIllegalValueException() {
         JsonAdaptedPerson person = new JsonAdaptedPerson(
```

Comment:
A bit random but, ok.

Reply:
included this in this branch by mistake while updating the automated tests

Reply:
yeye all's good with this specific part.